components/Features/Features.css

```diff
@@ -0,0 +1,5 @@
+.features {
+  padding-top: 0;
+}
+
+/*# sourceMappingURL=Features.css.map */
```

Comment:
Ohh, we should probably add .css files to `.gitignore` too since we don't need this compiled down.  I can add this.